config/crd/bases/gateway.kusk.io_staticroutes.yaml

```diff
@@ -79,6 +79,113 @@ spec:
                         description: OPTIONAL.
                         type: string
                     type: object
+                  jwt:
```

Comment:
Can you confirm this will not break static routes?

Reply:
Confirmed by changing `api/v1alpha1/staticroute_webhook.go` to only allow `auth.oauth2` to be specified. Example output:

```
$ kubectl apply -f ./
deployment.apps/static-route-example-1-top-level-upstream unchanged
service/static-route-example-1-top-level-upstream unchanged
Error from server: error when creating "api.yaml": admission webhook "vstaticroute.kb.io" denied the request: failed to validate options: `auth` in `StaticRoute` can only be `oauth2`: `jwt` has been specified
```